tweet-ui/src/main/java/com/twitter/sdk/android/tweetui/BaseTweetView.java

```diff
@@ -858,6 +858,16 @@ public void onPhotoClicked(MediaEntity mediaEntity) {
         return linkClickListener;
     }
 
+    /**
+     * Enable or disable Tweet actions
+     * @param listener Set the link click listener actions.
+     *                 If a tweet has links embedded you can listen for
+     *                 the clicks on the link and perform your own action.
+     */
+    public void setLinkClickListener(LinkClickListener listener) {
+        linkClickListener = listener;
```

Comment:
Just setting the `linkClickListener` member variable will not have the desired effect in all cases. The listener for `ClickableLinkSpan` is set when the Tweet is rendered. However, it is possible the listener is assigned after the text is rendered. Ideally we don't want to re-render the Tweet to set the click listener.